### PR TITLE
Added Springboot demo.

### PR DESCRIPTION
**Pull request by joerg84**

### CODE REVIEW

File: springboot/1.10/README.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
+# Springboot Demo
+
+This demo is all about deploying springboot java applications using the [dcos-maven plugin](https://github.com/dcos-labs/dcos-maven-plugin)
```
Comment (mattj-io) [inline springboot/1.10/README.md]: Should this be Spring Boot ? ie. capitalised and two words. Seems from https://projects.spring.io/spring-boot/ that is how they refer to it ..

File: springboot/1.10/README.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+- [Prerequisites](#prerequisites)
+- [Install](#install)
+- [Use the demo](#use)
+
+
+## Motivation
+
+This demo is about how to deploy a springboot demo using the [dcos-maven plugin](https://github.com/dcos-labs/dcos-maven-plugin). 
+
+Springboot is a great tool for developing Java based web applications profitting from the large spring ecosystem, including dependency injection, dynamically environment based configuration, rest, security, sql or nosql data stores.
```
Comment (mattj-io) [inline springboot/1.10/README.md]: As above, plus spring should be capitalised. rest, sql and nosql also should be capitalised.

File: springboot/1.10/README.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+The traditional options of deploying springboot applications are 
+* starting the fat jar directly on the machine
+* wrapping the application into a docker images.
+
+Both approaches have disadvantages:
+* Deploying the jars directly requires either manual mangement (imaggine restarts, scaling, updated) or some additional tooling.
```
Comment (mattj-io) [inline springboot/1.10/README.md]: typos - management, imagine

File: springboot/1.10/README.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+* Using Docker images introduces some overhead (imagine creating the container image, image size, fetching image, ...).
+
+DC/OS offers yet another option: using the [Universal Container Runtime](https://dcos.io/docs/1.10/deploying-services/containerizers/ucr/). Even though UCR also supports running Docker images, in this case we don't actually require a full Docker image as we have only two simple depencies:
```
Comment (mattj-io) [inline springboot/1.10/README.md]: typo, dependencies

File: springboot/1.10/README.md [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+* a JRE runtime
+* the application jar.
+
+With UCR we can simply fetch those to depencies and then run the jar.
```
Comment (mattj-io) [inline springboot/1.10/README.md]: typo, two dependencies

File: springboot/1.10/README.md [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+    {
+      "uri": "<MY JRE>",
+      "extract": true,
+      "executable": false,
+      "cache": false
+    }
+  ]
+  ```
+
+In this demo we will see how the dcos-maven plugin can automate the workflow of uploading jar and deploying the application from within Maven (and therefore our favorite IDE).
```
Comment (mattj-io) [inline springboot/1.10/README.md]: uploading the jar

File: springboot/1.10/README.md [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+
+* the artifact store to which we can upload the jar from our IDE so that DC/OS can download it.
+* the IDE project in which we develop our sample project.
+* the maven configuration for our cluster.
+
+### Artifact Store
+
+We need some artifact store which can serve the jar files. In this demo we decided to use
+Nexus as it is already available in the DC/OS package Catalog.
+So to install it we can simply use to UI and go to the Catalog and select `Nexus`.
```
Comment (mattj-io) [inline springboot/1.10/README.md]: simply use the UI, go to the Catalog

File: springboot/1.10/README.md [hunk 8]
```diff
@@ -0,0 +1,169 @@
...
+
+Next, we then select Configuration
+![Install Nexus](images/nexus_install.png)
+and configure Nexus to run only on the public agents, which will make it simpler to upload the jar files from our local IDE.
+![Configure Nexus](images/nexus_configure.png).
```
Comment (mattj-io) [inline springboot/1.10/README.md]: this link is broken, should be nexus-configure.png

Comment (joerg84) [reply springboot/1.10/README.md]: Renamed file

**Comment by unterstein** [comment]
amazing ❤️🎉

**Pull request merged**